src/MySql.DataLayer.Core/Repository/BaseMySqlRepository.cs

```diff
@@ -22,14 +22,18 @@ public abstract class BaseMySqlRepository<TEntity> : IMySqlDataRepository<TEntit
     {
```

Comment:
Here remove all `_databaseName`.
We already have the overload method passing the `MySqlConnection` in some methods.

Add the overload method passing the `MySqlConnection` for:
`GetAllAsync<TResult>`
`GetAllAsync`
`GetAsync`
`ExecuteStoredProcedure<TResult>`